programs/ssl/ssl_test_lib.c

```diff
@@ -455,7 +455,7 @@ static const struct {
     uint16_t tls_id;
     const char *name;
     uint8_t is_supported;
-} tls_id_curve_name_table[] =
+} tls_id_curve_group_table[] =
```

Comment:
Not there yet: `group_name`. A curve is a kind of group, and so is a set of ffdh parameters, so this table associates a group to its name. Should have "name" in its name, but no need for "curve" which is redundant with "group".